app/(landing)/Schedule.tsx

```diff
@@ -17,8 +17,8 @@ export default function Schedule() {
                     <ScheduleItem start="7:00 PM" end="7:30 PM">Coding Trivia Kahoot</ScheduleItem>
                     <ScheduleItem start="7:30 PM" end="8:45 PM">Discord Bots (Python + JS)</ScheduleItem>
                     <ScheduleItem start="8:45 PM" end="10:00 PM">Basic AI/ML</ScheduleItem>
```

Comment:
Probably say that this and the above are workshops? Same with Python / webscraping, Unity, and scientific computing.